cmd/applyconfig/applyconfig.go

```diff
@@ -388,6 +387,7 @@ func detectDryRunMethod(kubeconfig, context, username string) dryRunMethod {
 
 	var v ocVersionOutput
 	if err := json.Unmarshal(out, &v); err != nil {
+		logrus.WithError(err).Warning("Failed to detect dry run method from client and server versions; will continue with server dry-run")
```

Comment:
This fell through cracks in the original feature PR -- it's just a log, should be benign including it here too.